src/main/java/org/myrobotlab/service/Hd44780I2c.java

```diff
@@ -0,0 +1,242 @@
+package org.myrobotlab.service;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.myrobotlab.framework.Platform;
+import org.myrobotlab.framework.Service;
+import org.myrobotlab.framework.ServiceType;
+import org.myrobotlab.logging.Level;
+import org.myrobotlab.logging.LoggerFactory;
+import org.myrobotlab.logging.LoggingFactory;
+import org.slf4j.Logger;
+
+/**
+ * 
+ * HD44780 - Driver to play with lcd display panel
+ * Tested with HD44780 1602 panel, attached to pcf8574
+ * Driver pasted from Poduzov : https://github.com/Poduzov/PI4J-I2C-LCD
+ * 
+ * @author Moz4r
+ * 
+ */
+public class Hd44780I2c extends Service {

```

Comment:
Can we rename this class to remove the "i2c" from it.   Update the name to "HD44780"  seems a bit cleaner.

Reply:
yes, will do